packages/cli/src/metadataGeneration/exceptions.ts

```diff
@@ -11,7 +11,7 @@ export class GenerateMetadataError extends Error {
 }
 
```

Comment:
Small thing, do not crash while trying to make exceptions to throw (or when make warning messages).